pkg/capabilities/cli/cmd/generated_info.go

```diff
@@ -55,6 +56,9 @@ func generatedInfoFromSrc(src string, capID *string, typeInfo TypeInfo) (Generat
 
 	output := root.Outputs["Outputs"]
 
+	pkgParts := strings.Split(typeInfo.SchemaID, "/")
+	// skip http(s):// and drop the last part
+	fullPkg := strings.Join(pkgParts[2:len(pkgParts)-1], "/")
```

Comment:
This seems OK, but it is difficult to read. What about looking for the `//` and last `/` more directly?
```suggestion
	// skip http(s):// and drop the last part
	i := strings.Index(typeInfo.SchemaID, "//")
	j := strings.LastIndex(typeInfo.SchemaID, "/")
	fullPkg := typeInfo.SchemaID[i+1:j-1]
```
(this might have an off-by-one...)